### PR TITLE
create_appStor

### PR DESCRIPTION
**Pull request by elhadis**
This PR fixes the issue -Create app-store 

Fixes https://github.com/calcom/cal.com/issues/8617

to fix this issues I use yarn app-store create.

import examlpe from " /packages/app-store/index.ts

### CODE REVIEW

File: packages/app-store/example/index.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export * as api from "./api";
+require example :'./example';
```
Comment (Toledodev) [inline packages/app-store/example/index.ts:2]: Why this line is necessary?

File: packages/app-store/example/api/index.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export { default as add } from "./add";
+require example :'./example';
```
Comment (Toledodev) [inline packages/app-store/example/api/index.ts:2]: Why this line is necessary?

**Review by Toledodev — COMMENTED** [review]
Hey @elhadis, some questions please 😄

**Comment by elhadis** [comment]
Which line export or require?

**Comment by elhadis** [comment]
Export * mean all , and that allow us use
All the files or objects or function in folder app to use them in other
places.

**Comment by Toledodev** [comment]
@elhadis I was referring to another line (require L2)

**Pull request closed without merging**